test/test_span_token.py

```diff
@@ -107,6 +107,13 @@ def test_parse_children(self):
         child = next(iter(token.children))
         self._test_token(child, 'alt', src='src')
 
+    def test_angle_bracketed_inline_link_with_space(self):
```

Comment:
For the sake of simplicity and consistency, I suggest to shorten this to:

```python
    def test_angle_bracketed_inline_link_with_space(self):
        self._test_parse(span_token.Link, '[link](</my uri>)', 'link',
            target='/my uri', title='')
```

Reply:
yes, that's nicer.

Reply:
thanks